Doc/whatsnew/3.8.rst

```diff
@@ -690,6 +690,9 @@ Changes in Python behavior
 Changes in the Python API
 -------------------------
 
+* The function :func:`time.mktime` return type is now :class:`int`, rather than
```

Comment:
```suggestion
* The return type of :func:`time.mktime` is now :class:`int`, rather than
```

"The return type of the function" or "The function time.mktime's return type" are both also acceptable from a grammar standpoint.